functions/send_time_off_request_to_manager/mod.ts

```diff
@@ -50,7 +56,14 @@ export default SlackFunction(
     });
 
     if (!msgResponse.ok) {
-      console.log("Error during request chat.postMessage!", msgResponse.error);
+      logger.error(
+        `Error during request chat.postMessage! (error: ${msgResponse.error})`,
+      );
+      return {
```

Comment:
@filmaj Do you think we should not change the behavior for this error pattern? If so, I will revert this change.